spring-kafka/src/main/java/org/springframework/kafka/retrytopic/DestinationTopic.java

```diff
@@ -188,13 +196,15 @@ public Properties(Properties sourceProperties, String suffix, Type type) {
 		 * @param shouldRetryOn the exception classifications.
 		 * @param timeout the timeout.
 		 * @param autoStartDltHandler whether or not to start the DLT handler.
+		 * @param usedForExceptions the exceptions which destination is intended for
 		 * @since 2.8
 		 */
 		public Properties(long delayMs, String suffix, Type type,
```

Comment:
I suggest to have one more public ctor.
This way we won't have a breaking change in the point version.